plonky3/src/params/koala_bear.rs

```diff
@@ -0,0 +1,105 @@
+//! The concrete parameters used in the prover
+//! Inspired from [this example](https://github.com/Plonky3/Plonky3/blob/51c98987d1ee52c83a75142c1a2827d3ec71e563/keccak-air/examples/prove_koala_bear_poseidon2.rs)
+
+use lazy_static::lazy_static;
+
+use crate::params::{Challenger, FieldElementMap, Plonky3Field};
+use p3_challenger::DuplexChallenger;
+use p3_commit::ExtensionMmcs;
+use p3_dft::Radix2DitParallel;
+use p3_field::{extension::BinomialExtensionField, Field};
+use p3_fri::{FriConfig, TwoAdicFriPcs};
+use p3_koala_bear::{DiffusionMatrixKoalaBear, KoalaBear};
+use p3_merkle_tree::MerkleTreeMmcs;
+use p3_poseidon2::{poseidon2_round_numbers_128, Poseidon2, Poseidon2ExternalMatrixGeneral};
+use p3_symmetric::{PaddingFreeSponge, TruncatedPermutation};
+use p3_uni_stark::StarkConfig;
+
+use crate::params::poseidon2::{poseidon2_external_constants, poseidon2_internal_constants};
+
+use powdr_number::KoalaBearField;
+
+const D: u64 = 3;
```

Comment:
This is the only substantial difference from the BabyBear config. Because of the lower degree, Poseidon should be faster to evaluate and prove recursively.

Reply:
Could we at least document a single-character constant?